src/core/microservices/notifications.payload.builder.ts

```diff
@@ -57,6 +57,24 @@ export class NotificationsPayloadBuilder {
     return payload;
   }
 
+  async buildCommunityNewMemberPayload(userID: string, community: ICommunity) {
+    const payload = {
+      userID,
+      community: {
+        name: community.displayName,
+        type: community.type,
+      },
+      hub: {
```

Comment:
The user could also be joining a Challenge directly, so the Challenge ID also needs to be sent as part of the payload.

Reply:
The challenge info is sent correctly to the notification service - it is built in a later stage in `buildHubPayload` function.
![image](https://user-images.githubusercontent.com/7901834/164397506-be862495-6417-460a-aad8-4272f8e6f062.png)